test/e2e/node/pre_stop.go

```diff
@@ -177,7 +177,7 @@ var _ = SIGDescribe("PreStop", func() {
 		testPreStop(f.ClientSet, f.Namespace.Name)
 	})
 
-	ginkgo.It("graceful pod terminated should wait until preStop hook completes the process", func() {
+	ginkgo.It("should wait gracefully terminated pod, until preStop hook completes the process", func() {
```

Comment:
```suggestion
	ginkgo.It("should force gracefully terminated pod to wait, until preStop hook completes the process", func() {
```

What about this phrasing?

Reply:
+1.  I would also suggest removing "the process", and adding "a" in "force _a_ gracefully terminated pod" while we are add it.

Reply:
+1